### DISCUSSION TITLE
Add references to the plugin description in Installed/Available Plugins

### DISCUSSION THREAD

**Discussion by theDavidCoen** [btcpayserver/btcpayserver#3724] [Ideas / Feature Requests]
It would be useful to add references such as the link to the repo and the creator of the plugin to the description. 
This way we can have additional information and explore the code.
![Immagine 2022-05-06 114754](https://user-images.githubusercontent.com/38695835/167108839-b68609ff-b504-4846-a461-a995f9f26868.png)

**Comment by dstrukt** [comment]
Noted and agreed there needs to more information about the developer, code, etc.. and will definitely be something we address as we continue to improve the Plugins view!
